liota/core/package_manager.py

```diff
@@ -376,37 +359,67 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
-            if command in ["load", "unload", "delete", "reload", "update"]:
+            if command in ["load", "reload", "update"]:
                 #-----------------------------------------------------------
                 # Use these commands to handle package management tasks
 
                 with package_lock:
                     if len(msg) < 2:
                         log.warning("No package is specified: %s" % command)
                         continue
-                    if len(msg) > 2:
-                        list_packages = msg[1:]
+                    if len(msg) > 3:
+                        list_arg = msg[1:]
+                        list_packages = []
+                        cnt = len(list_arg)
+                        i = 0
+                        while (i < cnt):
+                            list_packages.append({list_arg[i]: list_arg[i+1]})
```

Comment:
Checksum check will fail, and that package will not be successfully loaded. It may affect other packages' loading since it may mess up the other package name and checksum. It will not be loaded until the correct inputs are given.